README.md

```diff
@@ -13,7 +13,16 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the early stages of development.
```

Comment:
```suggestion
Bevy is still in an experimental stage of development.
```

That is, I think we can find a better sentiment than "early" we can use here. I don't think experimental is definitely that though.

Perhaps:
- exploratory
- in flux
- (pre-teen)
- (ship-of-theseusing itself)
- (maximum warp)
- (code yeetage mode)

Reply:
```suggestion
Bevy is still under heavy development.
```